### ISSUE TITLE
Running problems with python 3.10

### ISSUE THREAD

**Issue by M1ch43lV** [bitromortac/lndmanage#138]
Hi! I need some help. I am using raspibolt. Created a separate user, gave access to lnd and 
installed the latest version as recommend in the installation guide. 
However I run into these problems. Any help / recommendation would be appreciated.

```
(venv) lndmanage@m2node:~$ lndmanage status
Traceback (most recent call last):
  File "/home/lndmanage/venv/bin/lndmanage", line 5, in <module>
    from lndmanage.lndmanage import main
  File "/home/lndmanage/venv/lib/python3.10/site-packages/lndmanage/lndmanage.py", line 18, in <module>
    from lndmanage.lib.fee_setting import FeeSetter, optimization_parameters
  File "/home/lndmanage/venv/lib/python3.10/site-packages/lndmanage/lib/fee_setting.py", line 12, in <module>
    from lndmanage.lib.forwardings import ForwardingAnalyzer
  File "/home/lndmanage/venv/lib/python3.10/site-packages/lndmanage/lib/forwardings.py", line 9, in <module>
    from lndmanage.lib.node import LndNode
  File "/home/lndmanage/venv/lib/python3.10/site-packages/lndmanage/lib/node.py", line 15, in <module>
    import lndmanage.grpc_compiled.lightning_pb2 as lnd
  File "/home/lndmanage/venv/lib/python3.10/site-packages/lndmanage/grpc_compiled/lightning_pb2.py", line 33, in <module>
    _descriptor.EnumValueDescriptor(
  File "/home/lndmanage/venv/lib/python3.10/site-packages/google/protobuf/descriptor.py", line 755, in __new__
    _message.Message._CheckCalledFromGeneratedFile()
TypeError: Descriptors cannot not be created directly.
If this call came from a _pb2.py file, your generated code is out of date and must be regenerated with protoc >= 3.19.0.
If you cannot immediately regenerate your protos, some other possible workarounds are:
 1. Downgrade the protobuf package to 3.20.x or lower.
 2. Set PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=python (but this will use pure-Python parsing and will be much slower).

More information: https://developers.google.com/protocol-buffers/docs/news/2022-05-06#python-updates
(venv) lndmanage@m2node:~$
```

**Comment by bitromortac** [comment]
I saw that problem also in the docker setup. I think a workaround could be to install an older version of protobuf (below version 4). If you could show me how you installed it, I'd guide you through it. The goal would be to install `python3 -m pip install "protobuf>=3.12,<4"`, but there are some dependencies which probably need to be downgraded as well.
Another way would be to put these versions into `setup.py`:
```
    install_requires=[
        "googleapis-common-protos>=1.52.0",
        "grpcio>=1.31.0",
        "grpcio-tools>=1.31.0",
        "networkx>=2.4",
        "numpy>=1.19.1",
        "protobuf==3.12.4",
        "Pygments>=2.7.4",
    ],
```
and then install into a new virtual environment with `pip install .`.

**Comment by M1ch43lV** [comment]
This helped! Thanks. Lndmanage is now working.

**Comment by bitromortac** [comment]
Fixed by #141.
